seacrowd/sea_datasets/udhr/udhr.py

```diff
@@ -0,0 +1,177 @@
+import os
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+from pathlib import Path
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Tasks, Licenses
+
+_CITATION = """\
+@misc{united1998universal,
+    title={The Universal Declaration of Human Rights, 1948-1998},
+    author={United Nations},
+    year={1998},
+    publisher={United Nations Dept. of Public Information New York}
+}
+"""
+_DATASETNAME = "udhr"
+_DESCRIPTION = """\
+The Universal Declaration of Human Rights (UDHR) is a milestone document in the history of
+human rights. Drafted by representatives with different legal and cultural backgrounds from
+all regions of the world, it set out, for the first time, fundamental human rights to be
+universally protected. The Declaration was adopted by the UN General Assembly in Paris on
+10 December 1948 during its 183rd plenary meeting.
+"""
+_HOMEPAGE = "https://unicode.org/udhr/index.html"
+_LICENSE = Licenses.UNKNOWN.value
+_URLS = "https://unicode.org/udhr/assemblies/udhr_txt.zip"
+
+_SUPPORTED_TASKS = [Tasks.SELF_SUPERVISED_PRETRAINING]
+_SOURCE_VERSION = "1.0.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+_LANGS = {
+    "khm": "Khmer",
+    "tha": "Thai",
+    "vie": "Vietnamese",
+    "ind": "Indonesian",
+    "zlm": "Malay",  # default mly_latn
+    "lao": "Lao",
+    "ban": "Bali",
+    "mya": "Burmese",
+    "ceb": "Cebuano",
+    "ace": "Aceh",
+    "tgl": "Tagalog",
+    # "abs": "",
+    "bug": "Bugis",
+    # "cja": "",
+    "cfm": "Chin, Falam",  # flm
+    "cnh": "Chin, Haka",
+    "ctd": "Chin, Tedim",
+    # "fil": "",
+    "hnj": "Hmong Njua",  # blu
+    # "iba": "",
+    # "dbj": "",
+    "jav": "Javanese",
+    "ilo": "Ilocano", 
+    "mnw": "Mon", 
+    "tet": "Tetun", 
+    "pam": "Pampangan", 
+    "lus": "Mizo", 
+    "min": "Minangkabau", 
+    "hni": "Hani", 
+    "shn": "Shan", 
+    "bcl": "Bicolano, Central", 
+    "hil": "Hiligaynon", 
+    "sun": "Sunda", 
+    "kkh": "Khun", 
+    "duu": "Drung", 
+    "tdt": "Tetun Dili", 
+    "mad": "Madura", 
+    "war": "Waray-waray", 
+    "blt": "Tai Dam", 
+    "hlt": "Chin, Matu",
+    "jav_java": "Javanese (Javanese)"
+}
+#Sort the keys
+_LANGS = dict(sorted(_LANGS.items()))
```

Comment:
Sorry this is not what I meant, I think we should sort the hardcoded list so that we can find the language easily as a code reader

Reply:
Oooh, okay